keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -262,36 +263,20 @@ Upgrade each node of the control plane to 1.22 with the following updates:
 - CCM deployed with `--enable-leader-migration`
 - CCM has its `--cloud-provider` set to the correct cloud provider
 
-After upgrade, CCM will run `route-controller`, `service-controller`, `cloud-node-controller`,
-and `cloud-nodelifecycle-controller`. The Leader Migration support will ensure CCM cleanly take out these controllers
-during the control plane upgrade.
+Starting from version 1.22, a proper default that represents the most recent migration will be provided.
+`*` in the `component` field indicates that the controllers can be run under either `kube-controller-manager` or `cloud-controller-manager`. Because the migration happens between
+the 1.21 `kube-controller-manager` with built-in cloud provider and 1.22 `cloud-controller-manager`, the controllers
+can run under either controller manager, which is exactly as described in the default configuration.
 
-As a reference, the provided default configuration in version 1.22 should have the `component` field of all affected
-controllers changed to `cloud-controller-manager`. The resulting default configuration should be equivalent to the
-following:
-
-```yaml
-kind: LeaderMigrationConfiguration
-apiVersion: controllermanager.config.k8s.io/v1alpha1
-leaderName: cloud-provider-extraction-migration
-resourceLock: leases
-controllerLeaders:
-  - name: route-controller
-    component: cloud-controller-manager
-  - name: service-controller
-    component: cloud-controller-manager
-  - name: cloud-node-controller
-    component: cloud-controller-manager
-  - name: cloud-nodelifecycle-controller
-    component: cloud-controller-manager
-```
-
-Please take note on how component names across both versions differs for each controller.
+During the upgrade, either KCM or CCM may hold the migration lease and thus run migrated controllers.
+However, at any moment, there is one and only one of them running the migrated controllers.
+After upgrade, only CCM will run `route-controller`, `service-controller`, `cloud-node-controller`, and 
```

Comment:
Remove cloud-node-controller and cloud-nodelifecycle-controller, replace them with 
"cloud-node-lifecycle" (? and "cloud-node").